Foundation/NSDictionary.swift

```diff
@@ -260,25 +260,25 @@ open class NSDictionary : NSObject, NSCopying, NSMutableCopying, NSSecureCoding,
     open var descriptionInStringsFileFormat: String {
         var lines = [String]()
         for key in self.allKeys {
-            let line = NSMutableString(capacity: 0)
-            line.append("\"")
+            var line = ""
+            line += "\""
             if let descriptionByType = getDescription(of: key) {
-                line.append(descriptionByType)
+                line += descriptionByType
             } else {
-                line.append("\(key)")
+                line += "\(key)"
             }
-            line.append("\"")
-            line.append(" = ")
-            line.append("\"")
+            line += "\""
+            line += " = "
```

Comment:
Given that we're appending these three constants together, is there a reason we don't just have them all in one constant `\" = \"` instead?